README.md

```diff
@@ -1,8 +1,8 @@
-# `kcp` provides a Kubernetes-like control plane with true multitenancy, with flexible compute powered by real Kubernetes clusters.
+# `kcp` provides a true multi-tenant Kubernetes control plane for workloads that span clusters
```

Comment:
"that span cluster" sounds wrong for me. That wording suggests federation as the focus which it isn't.

Reply:
I definitely wanted it to be vague here - because we are not implying M:1, nor 1:N, but M:N.  And we're starting with M:N, not M -> 2.  If you have a word that is vague like spans, please suggest.

Reply:
> ... for workloads that fill clusters

> ... for workloads across clusters

> ... for workloads on many clusters

Reply:
I went with option 3